test/integration_tests.jl

```diff
@@ -152,3 +168,23 @@
 
     _cleanup_old_branches(url)
 end
+
+@testset "$(service)" for service in [GITHUB, GITLAB]
+    personal_access_token = ENV["INTEGRATION_PAT_$(service)"]
+    test_repo = ENV["INTEGRATION_TEST_REPO_$(service)"]
+
+    api = get_api(service, personal_access_token)
+    user, _ = GitForge.get_user(api)
+    repo, _ = GitForge.get_repo(api, test_repo)
+
+    url = get_url(service, user, personal_access_token, test_repo)
+
+    env = Dict(
+        "$(service)_REPOSITORY" => test_repo,
+        "$(service)_TOKEN" => personal_access_token,
+        "CI_PROJECT_PATH" => "InveniaBot/compathelper_integration_test_repo.jl"
```

Comment:
```suggestion
        "CI_PROJECT_PATH" => INTEGRATION_TEST_REPO_GITLAB
```
I think this would make sense since it only affects the gitlab runs